### DISCUSSION TITLE
modeling degradation behavior in pybamm

### DISCUSSION THREAD

**Discussion by Thiyaga2025** [pybamm-team/PyBaMM#4246] [Q&A]
hi all,

I am trying to model battery degradation behavior in pybamm and facing some challenges. the test degradation curve shows an increase in capacity initially then starts decreasing continuously. i am not sure how to model such a trend in pybamm and which degradation mechanism to use. has any of you modeled similar trend. any help would be really appreciated in this regard. thanks.

best
Thiyaga

**Comment by brosaplanella** [comment]
I have never encountered this effect, so not sure what might cause it. Can you share a minimum working example that demonstrates this behaviour?

**Reply by Thiyaga2025** [reply]
hi Brosa,
thanks for the kind response. I got some cycle life information from the cell specification data sheet and using the same to fit my pybamm model. the test information(% discharge capacity retention vs cycles) shows an increase initially peaks at 105% (~250 cycles) and starts decreasing to ~95% at end of 1000 cycles. not sure this is an established behavior. just observed in this data set.

**Reply by brosaplanella** [reply]
Oh, yes, I think this is a fairly commonly observed feature in real data. I am not sure which models would capture this though.

**Reply by Thiyaga2025** [reply]
yeah, model is able to capture the quantity of capacity reduction but the trend is not as it is observed in the test.
